targets/TARGET_STM/can_api.c

```diff
@@ -335,7 +335,7 @@ int can_frequency(can_t *obj, int f)
     // !When the sample point should be lower than 50%, this must be changed to
     // !IS_FDCAN_DATA_TSEG2(ntq/nominalPrescaler), since
```

Comment:
Should this comment be updated ?

Reply:
I assume this shall be now `IS_FDCAN_NOMINAL_TSEG2` ?